site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/impl/SiteManageGroupSectionRoleHandler.java

```diff
@@ -1680,7 +1679,7 @@ public String processImportedGroups() {
 		
 		Set<Member> members= g.getMembers();
 		for(Member m: members) {
-			userIds.add(m.getUserId());
+			userIds.add(m.getUserEid());
```

Comment:
I think changing the breaks the lookup for existing site members. The `handler.lookupUsers(String)` method returns a `user ID` which it then looks for in the `existingUserIds` list that comes from this method. If this method returns `user EID`s then non of the existing users will match.

Reply:
The return value from handler.lookupUsers isn't used in the subsequent check of whether a user is existing or not so I have left the code as it was.  Testing the code gives the right results.